ROMFS/px4fmu_common/mixers/caipirinha_vtol_ailrev.main.mix

```diff
@@ -0,0 +1,16 @@
+#!nsh
+# Caipirinha vtol mixer for PX4FMU
+#
+#===========================
+R: 2- 10000 10000 10000 0
+
+#mixer for the elevons
+M: 2
+O:      10000  10000      0 -10000  10000
+S: 1 0  -10000  -10000      0 -10000  10000
```

Comment:
I don't think it's a good idea to have a separate autostart ID and mixer for my particular configuration in the main repo, but haven't decided whether I want to override the default mixer by maintaining my own mixer on an SD card.